Gems/Multiplayer/Code/Include/Multiplayer/IMultiplayer.h

```diff
@@ -102,10 +102,10 @@ namespace Multiplayer
         virtual void InitializeMultiplayer(MultiplayerAgentType state) = 0;
 
         //! Starts hosting a server.
-        //! @param port The port to listen for connection on
+        //! @param port The port to listen for connection on, 0 means use the currently configured port value of sv_port
```

Comment:
0 port has a very special meeting in networking and normally means dynamically go find me the port. sv_port as the fill in value seems less "dynamic"

Is there a potential for confusion here?

Reply:
Yes, but the problem here is port is a uint16_t and all values in that range are valid port numbers. I'd recommend we honor that usage of 0 if sv_port is also 0.